### ISSUE TITLE
Highlight integration documentation clarification

### ISSUE THREAD

**Issue by colorful-tones** [bylexus/grunt-md2html#6]
The existing Highlight.js integration documentation is confusing. How is the highlight.js included in the markdown files? I see the desired CSS being output by using the enabled: true and style: 'paraiso.dark', but do not see highlight.js being output anywhere in markup.

```
highlightjs: {
    enabled: true,
    style: 'paraiso.dark',
    compressStyle: true
},
```

Is user supposed to include the Highlight JS in their layout.html or Markdown somehow? , e.g. 
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.1.0/styles/default.min.css">
<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.1.0/highlight.min.js"></script>
<script src="<%= basepath %>/assets/highlight-init.js"></script>
```

**Comment by bylexus** [comment]
Dear Damon,

Thanks for your input, yes indeed you are right, one has to include the Highligh JS lib in the layout file. I will adapt the documentation accordingly.

**Comment by bylexus** [comment]
Dear Damon,

I just updated the package to 0.3.1, to support grunt >= 1.0 Beside that, I updated the documentation regarding highlightjs slightly. But I think the documentation was already clear before. Perhaps you just missed the relevant section, I will link it for you here. In short, you have 2 options to include highlightjs styles:

1. include a highlightjs CSS in your layout file manually
2. let grunt-md2html output a compressed version for you by using the `<%= highlightjs_style %>` template variable, as shown here:

https://github.com/bylexus/grunt-md2html#template-variables

Hope it becomes more clear now.
